### DISCUSSION TITLE
The media could not be loaded, either because the server or network failed or because the format is not supported.

### DISCUSSION THREAD

**Discussion by BidoSchiatti** [ant-media/Ant-Media-Server#5126] [Q&A]
![image](https://github.com/ant-media/Ant-Media-Server/assets/134113392/ef3d83bb-be00-478e-b337-dfff9704ac26)
I uploaded an mp4 video directly to ant media, but I can't run it, the following message appears, " The media could not be loaded, either because the server or network failed or because the format is not supported."

What could have happened?

**Comment by mekya** [comment]
Thank you for raising this issue @BidoSchiatti.

I think my colleague will respond to you soon. Btw, is there a chance that you send a sample file for us to confirm the problem?

**Comment by yashtandon113** [comment]
@BidoSchiatti 
Could you please confirm which version of Ant Media Server you are using? If it's v2.6.0, it's a known problem. 
Check out the following GitHub issue: https://github.com/ant-media/Ant-Media-Server/issues/5137

It will be fixed in v2.6.1. The release date for v2.6.1 is set for next week.

As a work around, you can play the uploaded stream using the Embed player with below format.
https://domain:5443/App-Name/play.html?id=vod_Id.mp4&playOrder=vod

You can also copy the URL from the Embed code of VOD.

If it is v2.5.3 or lower, we can investigate further if you can send us your sample file, as @mekya suggested.
